siatest/testnode.go

```diff
@@ -22,6 +30,165 @@ type TestNode struct {
 	primarySeed string
 }
 
+// DownloadToDisk downloads a previously uploaded file. The file will be downloaded
+// to a random location and returned as a TestFile object.
+func (tn *TestNode) DownloadToDisk(tf *TestFile, async bool) (*TestFile, error) {
```

Comment:
All of these helper functions should probably be split off into their respective files. So the download and upload stuff would go in renter.go, the mining stuff in miner.go, etc.